src/core/plugins/json-schema-2020-12-samples/fn/class/OptionRegistry.js

```diff
@@ -4,7 +4,17 @@
 import Registry from "./Registry"
 
 class OptionRegistry extends Registry {
-  #defaults = {}
+  #defaults = {
+    randomEnabled: false,
+    random: Math.random,
+    minInt: 0,
+    maxInt: 2147483647,
+    minLen: 4,
```

Comment:
```suggestion
    minLength: 0,
```

Reply:
@char0n this will generate samples with empty strings by default? Is that what we want as a default behavior?